index.js

```diff
@@ -179,6 +179,10 @@ const getSettings = function(options) {
   // get babel settings from the users provided function or use the defaults
   settings.babel = options.babel ? options.babel(defaultBabel()) : defaultBabel();
 
+  if (options.excludeCoverage) {
+    settings.excludeCoverage = options.excludeCoverage(settings.excludeCoverage);
```

Comment:
Previously we were running the user function and getting the result after we had already created the istanbul plugin. Which means that user options were ignored and only defaults were used.